[
    {
        "content": "<p>In one of my examples, I get the error \"Unable to find value set <a href=\"https://.\" target=\"_blank\" title=\"https://.\">https://.</a>..\" and I don't know where this comes from.<br>\nI declare the valueset, and the profile uses the value set as an example binding. My valueset has codes from snomed.<br>\nIn example 1, I populate code.coding.code but not code.coding.system, and it works ok<br>\nIn example 2, i populate code.coding.system as \"<a href=\"http://snomed.info/sct\" target=\"_blank\" title=\"http://snomed.info/sct\">http://snomed.info/sct</a>\" and it gives this error</p>",
        "id": 191354566,
        "sender_full_name": "Jose Costa Teixeira",
        "timestamp": 1584809038
    },
    {
        "content": "<p>does this mean that I must add the valueset to the terminology server (using <a href=\"http://tx.fhir.org\" target=\"_blank\" title=\"http://tx.fhir.org\">tx.fhir.org</a>), or...?</p>",
        "id": 191354622,
        "sender_full_name": "Jose Costa Teixeira",
        "timestamp": 1584809088
    },
    {
        "content": "<p><span class=\"user-mention\" data-user-id=\"191832\">@Jose Costa Teixeira</span> Just curious if you resolved this?  Since it's been a while I assume that you did, so then the question is what did the issue turn out to be and how did you solve it?  I've just run into what seems like is probably the same issue, for no really obvious reason that I can see at the moment.  I'm looking at rebuilding the package cache now.</p>",
        "id": 195852998,
        "sender_full_name": "Rob Hausam",
        "timestamp": 1588259310
    },
    {
        "content": "<p>That package cache doesn't seem to be the problem.</p>",
        "id": 195853100,
        "sender_full_name": "Rob Hausam",
        "timestamp": 1588259362
    },
    {
        "content": "<p>I think the issue is still there</p>",
        "id": 195853500,
        "sender_full_name": "Jose Costa Teixeira",
        "timestamp": 1588259523
    },
    {
        "content": "<p>OK.  If that's the case, then I guess we need to get it taken care of.  Do you have any further information or insights (even if no actual solution yet)?</p>",
        "id": 195853795,
        "sender_full_name": "Rob Hausam",
        "timestamp": 1588259602
    },
    {
        "content": "<p>I guess it does look like it's probably an issue with the terminology server for some reason - looking at the Tx log.  That may give me a clue for what to do - we'll see.</p>",
        "id": 195854229,
        "sender_full_name": "Rob Hausam",
        "timestamp": 1588259716
    },
    {
        "content": "<p>By any chance was your value set using 'exclude'?</p>",
        "id": 195854469,
        "sender_full_name": "Rob Hausam",
        "timestamp": 1588259805
    },
    {
        "content": "<p>the vs does not use exclude</p>",
        "id": 195856379,
        "sender_full_name": "Jose Costa Teixeira",
        "timestamp": 1588260532
    },
    {
        "content": "<p>the error i have is in this example:<br>\n<a href=\"http://build.fhir.org/ig/hl7-be/be-core/branches/master/AllergyIntolerance-noel-allergy.v1.html\" title=\"http://build.fhir.org/ig/hl7-be/be-core/branches/master/AllergyIntolerance-noel-allergy.v1.html\">http://build.fhir.org/ig/hl7-be/be-core/branches/master/AllergyIntolerance-noel-allergy.v1.html</a></p>",
        "id": 195856396,
        "sender_full_name": "Jose Costa Teixeira",
        "timestamp": 1588260540
    }
]